compiler/passes/scopeResolve.cpp

```diff
@@ -2304,17 +2394,16 @@ static void lookupUseImport(const char*           name,
             }
             if (sym) {
               if (sym->hasFlag(FLAG_PRIVATE) == true) {
-                if (sym->isVisible(context) == true &&
-                    isRepeat(sym, symbols)  == false) {
-                  symbols.push_back(sym);
-                  if (storeRenames && use->isARenamedSym(name)) {
+                if (sym->isVisible(context) == true) {
+                  bool added = saveSymbol(sym, symbols, foundSymsSet);
+                  if (added && storeRenames && use->isARenamedSym(name)) {
                     renameLocs[sym] = &use->astloc;
                   }
                 }
 
-              } else if (isRepeat(sym, symbols) == false) {
-                symbols.push_back(sym);
-                if (storeRenames && use->isARenamedSym(name)) {
+              } else {
+                bool added = saveSymbol(sym, symbols, foundSymsSet);
+                if (added && storeRenames && use->isARenamedSym(name)) {
```

Comment:
```suggestion
              } else if (saveSymbol(sym, symbols, foundSymsSet)) {
                if (storeRenames && use->isARenamedSym(name)) {
```

Reply:
I like the way I had it better because it explains what the result of `saveSymbol` means (it was added)

Reply:
I am not seeing any advantage to keeping the `else if` here, and it seems to be a purely stylistic choice.